content/lighthouse/configure-lighthouse-reports.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Configure Lighthouse Audits"
+date: 2020-09-23T15:29:49-04:00
+weight: 1
+---
+
+How to configure Lighthouse accessibility, SEO, and web-development best-practices audits against your Tugboat Previews
+
+- [Simple Lighthouse configuration](#simple-lighthouse-configuration)
+- [Use a custom Lighthouse configuration file](#use-a-custom-lighthouse-configuration-file)
+- [Specify Lighthouse settings per-URL](#specify-lighthouse-settings-per-url)
+- [Group Lighthouse reports via service alias](#group-lighthouse-reports-via-service-alias)
+
+{{% notice tier %}} This feature is only available to [Tugboat Enterprise](https://www.tugboat.qa/enterprise) or
+On-Premise subscribers. If you'd like to run Lighthouse audits against your Tugboat Previews,
+[contact us](https://www.tugboat.qa/contact/) to update your project to an Enterprise subscription.{{% /notice %}}
+
+If you want to generate Lighthouse reports for specific URLs, but turn visual diffs off for those urls, see:
+[Turn off visual diffs](/visual-diffs/configure-visual-diffs/#turn-off-visual-diffs).
+
+## Simple Lighthouse configuration
+
+To configure which pages get Lighthouse accessibility, SEO, and best practices audits, specify the _relative URLs_ of
+the pages in a `urls` key in the [service definition](/setting-up-services/). In this format, each URL can be a string,
+and you can list multiple urls.
+
+```yaml
+services:
+  apache:
+    urls:
+      # Conduct Lighthouse audits of the these URLs using the default options
+      - /
+      - /blog
+      - /about
+```
+
+## Use a custom Lighthouse configuration file
```

Comment:
The custom lighthouse config is not a reference to a file. The entire config object that would go into that file needs to be added to config.yml

Reply:
There are more references to a config file further down that will also need to be addressed